src/index.js

```diff
@@ -3,7 +3,8 @@
 import $ from 'jquery';
 import chai from 'chai';
 import testSuiteSkeleton from './assets/test-suite-skeleton';
-import mochaCSS from './assets/mocha-CSS';
+import testSuiteFCCStyles from 
+  '!style-loader!css-loader!./stylesheets/style.css';
```

Comment:
We still need to import the styles, but we don't need to append them to document.head

Reply:
Could you please put an inline comment about this to explain what it is doing? With a link to documentation would also be helpful.